src/seedu/addressbook/data/tag/Tagging.java

```diff
@@ -0,0 +1,15 @@
+package seedu.addressbook.data.tag;
+
+import seedu.addressbook.data.person.Person;
+
+public class Tagging {
```

Comment:
Missing header comment. All non trivial classes/methods should have javadoc format header comments.